### PR TITLE
Recent searches feature

### PR DESCRIPTION
**Pull request by sywhb**

### CODE REVIEW

**Comment by sywhb** [comment]
> Nice timing, just improving this on the client

Lol. Yeah, I saw that

File: packages/api/src/entity/search_history.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export class SearchHistory {
+  @PrimaryGeneratedColumn('uuid')
+  id!: string
+
+  @ManyToOne(() => User, { onDelete: 'CASCADE' })
+  @JoinColumn({ name: 'user_id' })
+  user!: User
+
+  @Column('varchar', { length: 255 })
+  keyword!: string
```
Comment (jacksonh) [inline packages/api/src/entity/search_history.ts:22]: Can we make this "term" ?

**Review by jacksonh — CHANGES_REQUESTED** [review]

File: packages/api/src/entity/search_history.ts (hunk 1 above)
Comment (sywhb) [reply packages/api/src/entity/search_history.ts:22]: Yes 👍 that is more accurate

**Comment by sywhb** [comment]
Hey @jacksonh this is ready for review

File: packages/api/src/entity/search_history.ts (hunk 1 above)
Comment (sywhb) [reply packages/api/src/entity/search_history.ts]: Updated

**Review by jacksonh — APPROVED** [review]

**Pull request merged**